airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -16,8 +17,9 @@ acceptance_tests:
   discovery:
     tests:
       - config_path: "secrets/config.json"
+         # refactored `spec`, but `app_id` is required for `1.2.0`
         backward_compatibility_tests_config:
-          disable_for_version: "0.2.33"
+          disable_for_version: "1.2.0"
```

Comment:
This section is for breaking changes in the schema. This schema change just adds some missing fields to the orders - what was noted as breaking here?

Reply:
the same comment as above about "app_id" filed.